Validation/RecoTrack/src/MTVHistoProducerAlgoForTracker.cc

```diff
@@ -2241,6 +2252,14 @@ void MTVHistoProducerAlgoForTracker::fill_simAssociated_recoTrack_histos(const H
   histograms.nSTRIPhits_vs_eta[count].fill(eta, tibHits + tidHits + tobHits + tecHits);
   histograms.nLayersWithMeas_vs_eta[count].fill(eta, track.hitPattern().trackerLayersWithMeasurement());
   histograms.nPXLlayersWithMeas_vs_eta[count].fill(eta, track.hitPattern().pixelLayersWithMeasurement());
+  if (doMTDPlots_) {
+    //  const auto mtdHits = track.hitPattern().numberOfValidTimingHits();
+    const auto btlHits = track.hitPattern().numberOfValidTimingBTLHits();
+    const auto etlHits = track.hitPattern().numberOfValidTimingETLHits();
+    histograms.nMTDhits_vs_eta[count].fill(eta, btlHits + etlHits);
```

Comment:
@mtosi sorry, the 3 histograms are identical in definition, just filled with either btl, etl or both. At the least the last histogram is redundant (or the first two). Could you please clarify the rationale behind this choice, that ahs an impact in memory?